### PR TITLE
Istio Benchmark Test

### PR DESCRIPTION
**Pull request by hisarbalik**
https://github.com/kyma-project/kyma/issues/4189

### CODE REVIEW

**Comment by hisarbalik** [comment]
#4189

File: performance-tools/istio-benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+It does so with a wide variety of payload sizes and connections.
+
+## Setup
+
+Before running the tests, create the pods needed. The `DNS_DOMAIN` can be `local`.
+
+```bash
+export NAMESPACE=twopods
+kubectl create namespace $NAMESPACE
+kubectl label namespace $NAMESPACE istio-injection=enabledl
```
Comment (clebs) [inline performance-tools/istio-benchmark/README.md]:
```suggestion
kubectl label namespace $NAMESPACE istio-injection=enabled
```

**Comment by tehcyx** [comment]
@clebs, @joek, @k15r, @venturasr, @nachtmaar, @rakesh-garimella, @lilitgh could you add your review for Hisar to merge this PR?

**Pull request closed without merging**